tests/test_amqps.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 
 import aio_pika
-from tests.test_amqp import TestCaseAmqp
+import tests
```

Comment:
I don't mind, just tell me why?

Reply:
As the commit message mentions: "Don't import TestCaseAmqp into global scope since pytest would detect it and run most tests in this file twice."
Less tests finishes faster.